toolsrc/src/vcpkg/commands.buildexternal.cpp

```diff
@@ -22,16 +22,16 @@ namespace vcpkg::Commands::BuildExternal
         const ParsedArguments options = args.parse_arguments(COMMAND_STRUCTURE);
 
         auto binaryprovider =
-            create_binary_provider_from_configs(paths, args.binarysources).value_or_exit(VCPKG_LINE_INFO);
+            create_binary_provider_from_configs(paths, args.binary_sources).value_or_exit(VCPKG_LINE_INFO);
 
         const FullPackageSpec spec = Input::check_and_get_full_package_spec(
             std::string(args.command_arguments.at(0)), default_triplet, COMMAND_STRUCTURE.example_text);
         Input::check_triplet(spec.package_spec.triplet(), paths);
 
-        auto overlays = args.overlay_ports ? *args.overlay_ports : std::vector<std::string>();
+        auto overlays = args.overlay_ports;
```

Comment:
```suggestion
        auto& overlays = args.overlay_ports;
```

Alternately just inline temp this thing now that it's no longer conditional?

Reply:
Since we take `args` as a const reference, we need to make a copy in order to modify in the next line.